src/main.rs

```diff
@@ -164,19 +193,23 @@ fn handle_suggestions(
         let mut i = 0;
         for solution in suggestion.solutions.iter() {
             println!("\n{}", solution.message);
+
+            // check whether we can squash all suggestions into a list
+            if solution.replacements.len() > 1 {
+                let first = solution.replacements[0].clone();
+                if solution.replacements.iter().all(|s| first.snippet.file_name == s.snippet.file_name && first.snippet.line_range == s.snippet.line_range) {
+                    prelude(&first);
+                    for suggestion in solution.replacements.iter() {
```

Comment:
Didn't clippy suggest `&solution.replacements` here?

Reply:
I think clippy throws up all kinds of things when used on rustfix. I never used it ;) I should. And I should feel bad for not having it done yet. ... Ah now I remember I wanted to fix rustfix with rustfix, but some clippy suggestions were breaking it. So I wanted to fix those first. I'm probably trapped in a loop